### ISSUE TITLE
Ability to force window scrollbars

### ISSUE THREAD

**Issue by johanwendin** [ocornut/imgui#476]
Greetings!

I would like to suggest an enhancement to ImGui.

There are times when it's desirable to always show scrollbars, for layouting and user experience.

I'm suggesting two more window flags are added (f.e. ForceHorizontalScrollbar, ForceVerticalScrollbar). If that route is taken, the scrollbars could easily be implemented by changing the following part of Begin: (unless I've missed something)

from:
```c++
window->ScrollbarY = (window->SizeContents.y > window->Size.y + style.ItemSpacing.y) && !(flags & ImGuiWindowFlags_NoScrollbar);

window->ScrollbarX = (window->SizeContents.x > window->Size.x - (window->ScrollbarY ? style.ScrollbarSize : 0.0f) - window->WindowPadding.x) && !(flags & ImGuiWindowFlags_NoScrollbar) && (flags & ImGuiWindowFlags_HorizontalScrollbar);
```

to:
```c++
window->ScrollbarY = (flags & ImGuiWindowFlags_ForceVerticalScrollbar) || ((window->SizeContents.y > window->Size.y + style.ItemSpacing.y) && !(flags & ImGuiWindowFlags_NoScrollbar));

window->ScrollbarX = (flags & ImGuiWindowFlags_ForceHorizontalScrollbar) || ((window->SizeContents.x > window->Size.x - (window->ScrollbarY ? style.ScrollbarSize : 0.0f) - window->WindowPadding.x) && !(flags & ImGuiWindowFlags_NoScrollbar) && (flags & ImGuiWindowFlags_HorizontalScrollbar));
```

**Comment by ocornut** [comment]
Done! Thanks.

**Comment by ocornut** [comment]
@johanwendin Unrelated to our other discussion, but I have just renamed those two flags.
`ImGuiWindowFlags_ForceVerticalScrollbar >>> ImGuiWindowFlags_AlwaysVerticalScrollbar`

The reasoning is 
- we already have a flag called _Always
- "force" sounds like an evil override-reasonable-behavior thing

It isn't listed in breaking changes because it appearing after 1.47 and is bound to be included in soon-1.48.
